operating_unit/security/ir.model.access.csv

```diff
@@ -0,0 +1,3 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_account_operating_unit_erp_manager,operating.unit erp.manager,model_operating_unit,base.group_erp_manager,1,1,1,1
+access_account_operating_unit_user,operating_unit user,model_operating_unit,,1,0,0,0
```

Comment:
@dreispt  Yes, just like in companies. Access to read operating units is universal. Record rules determine who can access what.